scribejava-apis/src/test/java/com/github/scribejava/apis/examples/XeroExample.java

```diff
@@ -0,0 +1,130 @@
+package com.github.scribejava.apis.examples;
+
+import java.util.Random;
+import java.util.Scanner;
+import com.github.scribejava.core.builder.ServiceBuilder;
+import com.github.scribejava.core.model.OAuth2AccessToken;
+import com.github.scribejava.core.model.OAuthRequest;
+import com.github.scribejava.core.model.Response;
+import com.github.scribejava.core.model.Verb;
+import com.github.scribejava.core.oauth.OAuth20Service;
+import java.io.IOException;
+import java.util.concurrent.ExecutionException;
+
+/*
+ * This full example needs a library to parse
+ * the JSON response and obtain a Xero Tenant Id
+ * in order to access protected resources.
+ * 
+ * If you add the following dependency, you can then
+ * uncomment the rest of the example code to access
+ * the Xero Organisation and other protected resources
+ * 
+ * <dependency>
+        <groupId>com.googlecode.json-simple</groupId>
+        <artifactId>json-simple</artifactId>
+        <version>1.1.1</version>
+    </dependency>
+ * 
+ */
```

Comment:
There is the com.fasterxml.jackson library as the dependency in the ScribeJava, you can use it.